cypress/e2e/ideyalabs/users.cy.js

```diff
@@ -0,0 +1,87 @@
+import ledgers from "../../support/fragments/finance/ledgers/ledgers";
+import topMenu from "../../support/fragments/topMenu";
+import users from "../../support/fragments/users/users";
+
+const RandomNumber = Math.floor(Math.random(9000) * 1000) + 1000;
```

Comment:
Either use `getRandomPostfix()` or if that doesn't meet your needs, add a new method `getRandomNumber()` in stripes-testing/cypress/support/utils/stringTools.js